DNN Platform/DotNetNuke.Abstractions/Logging/IEventLogService.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+namespace DotNetNuke.Abstractions.Logging
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// The event log service provides APIs for
+    /// managing existing Event Logs in the database.
+    /// </summary>
+    public interface IEventLogService
```

Comment:
@ahoefling you had talked about `IEventLogService` implementing `IEventLogger` also.  Did I misunderstand, or was that just missed?

Reply:
That was missed and I think still think the `IEventLogService` should implement `IEventLogger`. This will simplify integrations to only have 1 dependency if the code needs to log and manage the log table